### PR TITLE
Slides

### LINKED ISSUE #52: Initital Slides
First draft of slides

### PR DESCRIPTION
**Pull request by cristeahub**
This fixes #52

### CODE REVIEW

**Comment by sigvef** [comment]
image
 does this mean you are reviewing, @aleksanb?

File: slides/index.html [hunk 1]
```diff
@@ -20,11 +20,23 @@
...
+
+- Først og fremst takk til fagkom for gratis muffins og kaffe
+- 4.-5. klasse data@NTNU
+- Blåtannfrelste personligheter
+- Laget et open source blåtannbibliotek for mbed i forbindelse med sommerjobb hos Nordic Semiconductor
+
+---
+class: center, middle
+
+## Hvem var Harald Blåtann?
```
Comment (sigvef) [inline slides/index.html:33]: Maybe better with just "Harald Blåtann", and then a huge fullscreened background image of him looking all fierce-like.

File: slides/index.html [hunk 2]
```diff
@@ -20,11 +20,23 @@
 
 ## Hvem er vi?
 
-Studerer datateknikk på NTNU
+???
+
+- Først og fremst takk til fagkom for gratis muffins og kaffe
```
Comment (stianjensen) [inline slides/index.html]: Should we say 'thanks for the muffins' but not 'free'? :)

Comment (stianjensen) [inline slides/index.html]: Also, we will be using these slides with tihlde, too. So we'll have to remember to update this part

File: slides/index.html [hunk 3]
```diff
@@ -20,11 +20,23 @@
...
+---
+class: center, middle
+
+## Hvem var Harald Blåtann?
 
 ???
 
-4.-5. klasse
+- Konge av Danmark(~958-986) og Norge(~970-975/986)
+- Samlet alle Danske stammer til ett forent rike og introduserte kristendom
```
Comment (sigvef) [inline slides/index.html:40]: two or three slides with a bit more detailed story-telling here? maps, illustrations etc?

File: slides/index.html (hunk 1 above)
Comment (sigvef) [inline slides/index.html:33]: Would be cool to have a fun fact here: why is he called bluetooth? did he have a rotten tooth?

File: slides/index.html [hunk 4]
```diff
@@ -20,11 +20,23 @@
...
+- Oppgavet til navnet Bluetooth da denne teknologien skal samle alle kommunikasjonsprotokoller til én standard
```
Comment (sigvef) [inline slides/index.html]: Opphavet

File: slides/index.html [hunk 5]
```diff
@@ -32,49 +44,108 @@
...
+- Starter med å annonsere at den finnes -> med info som navn, adresse og liknende
+- Når noen kopler seg til forteller den hva den kan tilby (hvilke services den har)
 
 ---
-class: center
+class: center, middle
 
 # mbed
 
+Lavterskeltilbud til lavlevelprogrammering
```
Comment (stianjensen) [inline slides/index.html]: 'lavlevel' is an awkward combo of norwegian and english. either 'lavnivåprogrammering' or 'low level programming'

File: slides/index.html [hunk 6]
```diff
@@ -32,49 +44,108 @@
...
+Trådløs teknologi  
+
+Kort rekkevidde  
+
+Sender data mellom enheter
 
 ???
 
-Få med det mest grunnleggende
+- Bluetooth er en kommunikasjonsprotokoll for å kommunisere mellom enheter
```
Comment (stianjensen) [inline slides/index.html]: communicate to communicate. Maybe just 'protokoll for kommunikasjon mellom enheter'

File: slides/index.html (hunk 4 above)
Comment (stianjensen) [inline slides/index.html]: 'Opphavet'?

File: slides/index.html [hunk 7]
```diff
@@ -32,49 +44,108 @@
...
+- Vi har en nettside, som inneholder **guider**, **oppgavene**, **slides** osv
+- Bruk denne som ressurs når du jobber
+- Denne er tilgjengelig også etter kurset så den kan brukes om du trenger hjelp senere
 
 ---
-class: center
+class: center, middle
 
-# Bluetooth
+# Blåtann
```
Comment (sigvef) [inline slides/index.html]: Mention that we're doing BLE and not regular Bluetooth perhaps? ("what is the difference between gatt/service-oriented bluetooth and the regular kind that is in the douche-headset?")
image

File: slides/index.html [hunk 8]
```diff
@@ -32,49 +44,108 @@
...
 # C++
 
+Utviklingsspråket til mbed  
+
+Høynivåspråk i embedded-verden
+
+???
+
+- (Håndsopprekning) Hvor mange har jobba i C++ før?
+- Det likner på Java
```
Comment (sigvef) [inline slides/index.html]: Maybe like if Java and C had a baby?

File: slides/index.html (hunk 2 above)
Comment (cristeahub) [inline slides/index.html]: Yes. If we forget this, it's thankfully only in the presenter notes

**Comment by aleksanb** [comment]
> does this mean you are reviewing, @aleksanb?

Correctemundo, but more eyes are better! 
I saw stian using it just now, and i thought i'd like to try that also

File: slides/index.html (hunk 2 above)
Comment (sigvef) [inline slides/index.html]: "We're giving it away, but it is most certainly not free!"
image

File: slides/index.html (hunk 4 above)
Comment (sigvef) [inline slides/index.html]: s/Oppgavet/Opphavet/

**Comment by sigvef** [comment]
Looks good so far! Let's get more large and nice images in the slides to help bring our points across.

File: slides/index.html [hunk 9]
```diff
@@ -32,49 +44,108 @@
...
 
 ## 3 timer
+
+Blåtann  
+mbed  
+Hjelpemidler
+
 ## 3 oppgaver
 
+Lag en mbed med blåtann
```
Comment (aleksanb) [inline slides/index.html:56]: Should this rather be "Lag en mbed som tror den er et ibeacon, eller noe i den duren?"

File: slides/index.html [hunk 10]
```diff
@@ -32,49 +44,108 @@
...
 
 ---
-class: center
+class: center, middle
 
 # mbed
 
+Lavterskeltilbud til lavlevelprogrammering  
+
+Utviklingsmiljøet i nettleseren
```
Comment (aleksanb) [inline slides/index.html:104]: Burde det vært "Utviklingsmiljø i nettleseren"?

**Pull request merged**